packages/api/src/external/ehr/elation/command/sync-patient.ts

```diff
@@ -173,3 +167,58 @@ async function createElationPatientLink({
     });
   }
 }
+
+async function createElationPatientMetadata({
+  cxId,
+  elationPracticeId,
+  elationPatientId,
+  metriportPatientId,
+  elationApi,
+}: {
+  cxId: string;
+  elationPracticeId: string;
+  elationPatientId: string;
+  metriportPatientId?: string;
+  elationApi?: ElationApi;
+}): Promise<void> {
+  const ehrDashUrl = await createElationPatientLink({ elationPracticeId, elationPatientId });
+  const api = elationApi ?? (await createElationClient({ cxId, practiceId: elationPracticeId }));
+  await api.updatePatientMetadata({
+    cxId,
+    patientId: elationPatientId,
+    metadata: {
+      object_id: metriportPatientId ?? "unknown",
+      object_web_link: ehrDashUrl,
+    },
+  });
+}
+
+export async function updateOrCreateElationPatientMetadata({
```

Comment:
nit: usually we name these `createOrUpdate...`, not sure if there's a specific reason to name it differently though